src/BizHawk.Emulation.Cores/Computers/DOS/DOSBox.ICycleTiming.cs

```diff
@@ -0,0 +1,15 @@
+﻿using BizHawk.Emulation.Common;
+
+namespace BizHawk.Emulation.Cores.Computers.DOS
+{
+	public partial class DOSBox : ICycleTiming
+	{
+		// DOSBox emulates the internal core at stable 1ms increments, regardless of the configuration. Therefore, this is basis for time calculation.
+		double ICycleTiming.ClockRate => 1000.0;
+
+		// How many ms have elapsed in the previous emulated frame

```

Comment:
Does this mean "during the previous frame only" or " overallup to the current frame overall"?

Reply:
Looking at the implemention this seems to be the ticks from one frame only, which is not what the frontend expects.